Subject: [PATCH] d
---
Index: src/CoinUtilsConfig.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/CoinUtilsConfig.h b/src/CoinUtilsConfig.h
--- a/src/CoinUtilsConfig.h	(revision a6385801e71b2d060198d88f3a09bd9046fe8de6)
+++ b/src/CoinUtilsConfig.h	(revision 4e484880d4decffd033d994e303886f0afdb5ed7)
@@ -33,6 +33,11 @@
  * we want it to be __attribute__((__visibility__("default"))) when building with GCC,
  *   so user can compile with -fvisibility=hidden
  */
+#define XSTR(x) STR(x)
+#define STR(x) #x
+
+#pragma message(XSTR(_WIN32) " " XSTR(DLL_EXPORT))
+
 #ifdef DLL_EXPORT
 #undef COINUTILSLIB_EXPORT
 #define COINUTILSLIB_EXPORT __declspec(dllexport)
